docs/commands/data.json

```diff
@@ -499,19 +505,19 @@
+    "markdown": "## Description\n\nSPK command to set a ring as the default for an initialized bedrock project.\n\n## Example\n\nFor a bedrock.yaml file that looks like this:\n\n```yaml\nrings:\n  dev:\n    isDefault: true\n  qa:\n  prod:\nservices:\n  - path: ./\n    displayName: \"fabrikam\"\n    helm:\n      chart:\n        branch: master\n        git: \"https://dev.azure.com/fabrikam/frontend/_git/charts\"\n        path: frontend\n    k8sBackend: \"fabrikam-k8s-svc\"\n    k8sBackendPort: 80\n    middlewares: []\n    pathPrefix: \"fabrikam-service\"\n    pathPrefixMajorVersion: \"v1\"\nvariableGroups:\n  - fabrikam-vg\n```\n\nrunning `spk ring set-default prod` will result in:\n\n1. `prod` will be set as the default in `bedrock.yaml`:\n\n   ```yaml\n   rings:\n   dev:\n   qa:\n   prod:\n     isDefault: true\n   services:\n     - path: ./\n       displayName: \"fabrikam\"\n       helm:\n       chart:\n         branch: master\n         git: \"https://dev.azure.com/fabrikam/frontend/_git/charts\"\n         path: frontend\n       k8sBackend: \"fabrikam-k8s-svc\"\n       k8sBackendPort: 80\n       middlewares: []\n       pathPrefix: \"fabrikam-service\"\n       pathPrefixMajorVersion: \"v1\"\n   variableGroups:\n     - fabrikam-vg\n   ```\n\n2. Commiting these changes will trigger the project's lifecycle pipeline, which\n   may update the default \"no-ring\" IngressRoutes to route to the `prod` service\n   in the linked HLD repository,\n   [pending the work in this issue](https://github.com/microsoft/bedrock/issues/1084).\n"
```

Comment:
trying to get rid of these changes from my branch but the pre-commit hook keeps adding it back

Reply:
you should not touch this file. the content of this file is generated

Reply:
I didn’t, it automatically added these changes to this file